internal/test/e2e/e2e_test.go

```diff
@@ -153,18 +148,53 @@ func installOperator(t *testing.T, kubectl *kubectlContext, kind *cluster.Contex
 	err = kubectl.Apply("--kustomize", filepath.Join(*fRepoRoot, "config", "crd"))
 	require.NoError(t, err)
 
-	imageFile, err := os.Open(imageTar)
-	require.NoError(t, err)
-	defer func() {
-		assert.NoError(t, imageFile.Close(), "failed to close operator image tar")
-	}()
-	// Load the built image into the Kind cluster.
-	t.Log("Loading image in to Kind cluster")
 	nodes, err := kind.ListInternalNodes()
 	require.NoError(t, err)
+
+	// When we upload images to the Kind nodes, we also touch a file with name
+	// equal to the Docker image ID.
+	// If the image ID file is not present on one or more of the Kind nodes, we
+	// perform the slow process of exporting the saving Docker image to a local
+	// file and then uploading it to each Kind node.
+	// Additionally, if the image does change then we force re-deploy the
+	// operator.
+	// If the image ID file is present, we can skip that process and get
+	// straight to running the tests.
+	imageMarkerPath := "/tmp/" + imageID
+	uploadImages := false
 	for _, node := range nodes {
-		err := node.LoadImageArchive(imageFile)
+		err := node.Command("ls", imageMarkerPath).Run()
```

Comment:
would it be more reliable to run some `ctrd` command to check if the image is loaded? dont know if this is possible

Reply:
Yes, this is a bit of a hack and there is some existing logic in the `kind load image` command which I can re-use.

Reply:
I've modified it to use `ctr -n k8s.io images ls name=sha256:aaabbbbccc` which seems to work.